map/traffic_manager.cpp

```diff
@@ -120,8 +122,10 @@ void TrafficManager::Clear()
 {
   m_currentCacheSizeBytes = 0;
   m_mwmCache.clear();
-  m_lastMwmsByRect.clear();
-  m_activeMwms.clear();
+  m_lastDrapeMwmsByRect.clear();
+  m_lastRoutingMwmsByRect.clear();
```

Comment:
I agree that it's not a good approach and see copy-past like code your talking about, but I propose to leave it as is for now.